projects/plugins/jetpack/extensions/blocks/subscriptions/view.js

```diff
@@ -6,7 +6,48 @@ import '../../shared/memberships.scss';
 import domReady from '@wordpress/dom-ready';
 import { handleIframeResult } from '../../../extensions/shared/memberships';
 
+// @ts-ignore
+function show_iframe_retrieve_subscriptions_from_email() {
+	const form = document.querySelector( '.wp-block-jetpack-subscriptions__container form' );
```

Comment:
Note that you might have multiple subscribe forms on the page simultaneously, and you might end up picking the email input from wrong one.

You could generate the whole subscribe block container an unique IDs across sessions, and rely on that.

Reply:
I agree. But I am using the same logic as the "payment" part. Not perfect, and needs improvement. Yes, like every bit of code on Earn...

Reply:
The line cause bugs which were fixed in https://github.com/Automattic/jetpack/pull/33932

Reply:
I took that into account and that was fixed in https://github.com/Automattic/jetpack/pull/33800/files#diff-364058ace64d8c69cdbbac7d7cc6afd5116da163ce99d8e2928b5a5194de4487R14, but you do not want that approach.